src/pages/governance/_components/ProposalTable.tsx

```diff
@@ -133,6 +139,18 @@ function ProposalRow({
       <OverflowTable.Cell className="align-middle text-gray-900 dark:text-dark-gray-900">
         {ProposalDisplayName[proposal.type]}
       </OverflowTable.Cell>
+      <OverflowTable.Cell className="align-middle text-gray-900 dark:text-dark-gray-900">
+        {proposal.type === GovernanceProposalType.VOTE_OF_CONFIDENCE ? (
+          "N/A"
+        ) : (
+          <NumericFormat
+            value={new BigNumber(proposal.amount ?? 0).toFixed(2)}
+            fixedDecimalScale
```

Comment:
i think this would only work if we defined `decimalScale`, let's not add this since we have `toFixed(2)`